aspnetcore/blazor/components/js-spa-frameworks.md

```diff
@@ -139,6 +139,34 @@ Quote &copy;1988-1999 Satellite of Love LLC: [*Mystery Science Theater 3000*](ht
 > rootComponent.dispose();
 > ```
 
+The preceding example dynamically renders the root component when the `showQuote()` JS function is called. To render a root component into a container element when Blazor starts, use a [JavaScript initializer](xref:blazor/fundamentals/startup#javascript-initializers) to render the component, as the following example demonstrates.
+
+The following example builds on the preceding example, using the `Quote` component, the root component registration in `Program.cs`, and the initialization of `jsComponentInitializers.js`. The `showQuote()` function (and the `script.js` file) aren't used.
+
+In HTML, place the target container element, `quoteContainer2` for this example:
+
+```html
+<div id="quoteContainer2"></div>
+```
+
+In a [JavaScript initializer](xref:blazor/fundamentals/startup#javascript-initializers). The following example only handles pasting text, but you could use arbitrary JavaScript APIs to deal with users pasting other types of data, such as images.
```

Comment:
> In a JavaScript initializer.

I'm guessing something got left out of this sentence. Or am I misinterpreting it?

> The following example only handles pasting text, but you could use arbitrary JavaScript APIs to deal with users pasting other types of data, such as images.

Forgive me if I'm missing something, but how does the example relate to pasting?

Reply:
Yes, good 👁️ ... This paragraph must be a stray bad paste 😈 while I was working on the text. The "paste" part refers to the custom paste example in the *Event Handling* article. This paragraph shouldn't be here. I'll 🔪 it out on the next commit.

Reply:
Actually ... I don't think it's a stray paste ... I think it's just unfinished business. It should say ...

> Using a \[JavaScript initializer](xref:blazor/fundamentals/startup#javascript-initializers), add the root component to the target container element.

I'll go with that.